frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts

```diff
@@ -773,6 +774,60 @@ describe('Serving Runtime List', () => {
         .should('have.text', '3');
     });
 
+    it('Successfully deletes model server', () => {
+      initIntercepts({
+        projectEnableModelMesh: true,
+        disableKServeConfig: false,
+        disableModelMeshConfig: true,
+        inferenceServices: [
+          mockInferenceServiceK8sResource({ name: 'test-inferebce', isModelMesh: true }),
+          mockInferenceServiceK8sResource({
+            name: 'ovms-testing',
+            displayName: 'OVMS ONNX',
+            isModelMesh: true,
+          }),
+        ],
+      });
+      cy.intercept(
+        {
+          method: 'DELETE',
+          pathname:
+            '/api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes/test-model-legacy',
+        },
+        {},
+      ).as('deleteServingRuntimes');
+      cy.intercept(
+        {
+          method: 'DELETE',
+          pathname: '/api/k8s/api/v1/namespaces/test-project/serviceaccounts/test-model-legacy-sa',
+        },
+        {},
+      ).as('deleteServiceAccounts');
+      cy.intercept(
+        {
+          method: 'DELETE',
+          pathname:
+            '/api/k8s/apis/rbac.authorization.k8s.io/v1/namespaces/test-project/rolebindings/test-model-legacy-view',
+        },
+        {},
+      ).as('deleteRoleBindings');
+      projectDetails.visit('test-project');
+      modelServingSection
+        .getModelMeshRow('ovms')
+        .find()
```

Comment:
A small change here, Can you remove find()? This is not needed as findKebabAction is inherited from tableRow class